README.md

```diff
@@ -1,3 +1,43 @@
-# flatpak-platform
+<div align="center">
+  <h1 align="center"><center>flatpak-platform</center></h1>
+  <h3 align="center"><center>An elementary OS Flatpak platform</center></h3>
+  <br>
+  <br>
+</div>
 
-A platform for Flatpak from elementary OS.
+<p align="center">
+  <img src="https://github.com/elementary/flatpak-platform/workflows/Daily/badge.svg" alt="Daily">
+</p>
+
+---
+
+## Building
+
+If you want to build this flatpak platform locally, you will need the flathub flatpak repository installed. You can
+follow instructions for that [here](https://flatpak.org/setup/).
+
+Next you will need:
+
+- A lot of space (~ 15 Gb)
+- `flatpak-builder`
+- `git`
+
+And finally, you can run this command to build and install:
+
+```sh
+flatpak-builder --force-clean --install-deps-from=flathub --ccache --repo=elementary --install builddir ./io.elementary.Sdk.json
+```
+
+## Publishing
+
+This repository has CI/CD setup. Building and publishing is handled automatically with the help of some amazing GitHub
+actions, and takes no human interaction.
+
+### Daily
+
+Every push to the `main` branch will start a build and publish it to our repository under the `daily` branch. This
+branch is build frequently and is **not recommended for production**.
+
+### Stable
+
+**TODO**
```

Comment:
Still not 100% sure on how we will do this. It will probably involve running an action on release tagging, string replacing the manifest version with that tag, then publishing.